python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -387,6 +393,57 @@ function AIConfigEditorBase({
     );
   }, [updateModelCallback]);
 
+  const onUpdatePromptMetadata = useCallback(
+    async (promptId: string, newMetadata: JSONObject) => {
+      if (!debouncedUpdatePrompt) {
+        // Just no-op if no callback specified. We could technically perform
+        // client-side updates but that might be confusing
+        return;
+      }
+
+      dispatch({
+        type: "UPDATE_PROMPT_METADATA",
+        id: promptId,
+        metadata: newMetadata,
+      });
+
+      const onError = (err: unknown) => {
+        const message = (err as RequestCallbackError).message ?? null;
+        showNotification({
+          title: "Error updating prompt name",
```

Comment:
Typo: should be metadata instead of name

Reply:
Thanks, good catch